Cannot build the adapter: Could not find /dune.module

## Description

I am building the adapter for the first time. I got a fresh copy of `installdumux.py` from the GitLab of DuMuX 3.8 and I have placed the dumux-adapter in the same directory. However, when I try to build it, I get an error that sounds like I have not set some path (otherwise, why the starting slash?):

```
~/software/dumux$ ls
dumux  dumux-adapter  dune-common  dune-geometry  dune-grid  dune-istl  dune-localfunctions
~/software/dumux$ ./dune-common/bin/dunecontrol --only=dumux-precice all
--- going to build dumux-precice  ---
--- calling all for dumux-precice ---

ERROR: could not find /dune.module
--- Failed to build dumux-precice ---
Terminating dunecontrol due to previous errors!
```

```
~/software/dumux/dune-common [releases/2.9]$ ls
bin           CMakeLists.txt   COPYING  dune-common.pc.in  .gitignore      lib         .mailmap        README.md  share
CHANGELOG.md  config.h.cmake   doc      dune.module        .gitlab-ci.yml  LICENSE.md  pyproject.toml  .reuse     TODO
cmake         CONTRIBUTING.md  dune     .git               INSTALL         LICENSES    python          setup.py

```

@mathiskelm @Fujikawas @ajaust @IshaanDesai any clue what I have forgotten?

I think I have followed all instructions from the docs: https://precice.org/adapter-dumux-get.html

### Environment

Ubuntu 22.04.

After setting:

```
export DUNE_CONTROL_PATH=/home/gc/software/dumux:/home/gc/software/dumux/dumux-adapter:${DUNE_CONTROL_PATH}
# /home/gc/software/dumux:/home/gc/software/dumux/dumux-adapter:/home/gc/software/dune
```

I get:

```
~/software/dumux$ ./dune-common/bin/dunecontrol --only=dumux-precice all
ERROR: multiple definition of module dune-istl
previous defined in:
  /home/gc/software/dumux/dune-istl/dune.module
redefined in:
  /home/gc/software/dune/dune-istl/dune.module
Execution of dunecontrol terminated due to errors!

```

--- a/docs/get.md
+++ b/docs/get.md
@@ -77,6 +77,11 @@ Note that to change this setting it may be required to clear the CMake caches in
 
 For more ways to manipulate/adapt the build and configuration step, please consult the `dunecontrol` documentation.
 
+{% important %}
+In case you are building both the [DUNE adapter](https://precice.org/adapter-dune.html) and the DuMux adapter, make sure that you build all modules with the same CMake options.
+To ensure consistency among DUNE module versions, you might want to explicitly download and build each module, instead of using the `installdumux.py` script.
+{% endimportant %}
+
 ## Build tests (optional)
 
 Optional, but recommended: Build all tests to verify the installation. For this navigate in the `build-cmake/` directory and build the `build_tests` target.

--- a/docs/get.md
+++ b/docs/get.md
@@ -78,8 +78,10 @@ Note that to change this setting it may be required to clear the CMake caches in
 For more ways to manipulate/adapt the build and configuration step, please consult the `dunecontrol` documentation.
 
 {% important %}
-In case you are building both the [DUNE adapter](https://precice.org/adapter-dune.html) and the DuMux adapter, make sure that you build all modules with the same CMake options.
+In case you want to get both the [DUNE adapter](https://precice.org/adapter-dune.html) and the DuMux adapter, add all modules in the same directory, and use
+`dunecontrol` to build all modules.
 To ensure consistency among DUNE module versions, you might want to explicitly download and build each module, instead of using the `installdumux.py` script.
+Finally, make sure that you build all modules with the same CMake options (use the same `--opts` for both, or ommit it to use the default CMake options from vanilla DUNE).
 {% endimportant %}
 
 ## Build tests (optional)

